doc/rst/platforms/infiniband.rst

```diff
@@ -25,21 +25,44 @@ Chapel with InfiniBand support, set:
       export CHPL_COMM=gasnet
       export CHPL_COMM_SUBSTRATE=ibv
 
-Alternatively, when running on a Cray CS or HPE Apollo system
-``CHPL_HOST_PLATFORM`` can instead be set, in which case the comm
+Alternatively, when running on an HPE Apollo or Cray CS system
+``CHPL_HOST_PLATFORM`` can be set instead, in which case the comm
 and substrate settings will be inferred.
 
+For HPE Apollo:
+
+.. code-block:: bash
+
+      export CHPL_HOST_PLATFORM=hpe-apollo
+
 For Cray CS:
 
 .. code-block:: bash
 
       export CHPL_HOST_PLATFORM=cray-cs
 
-For HPE Apollo:
+
+UCX alternative
+---------------
+
+In rare cases, we have had users who are unable to use the ``ibv``
+conduit on their InfiniBand systems.  In such cases, the ``ucx``
+conduit is an alternative to consider, which requires setting:
 
 .. code-block:: bash
 
-      export CHPL_HOST_PLATFORM=hpe-apollo
+      export CHPL_COMM=gasnet
+      export CHPL_COMM_SUBSTRATE=ibv
+
+(and, most likely, ``export CHPL_LAUNCHER=gasnetrun_ucx`` when you
+reach the following section).
```

Comment:
`CHPL_LAUNCHER=gasnetrun_ucx` should be inferred when `CHPL_COMM_SUBSTRATE=ucx` , but no harm leaving this